atlascope/core/management/populate/investigations.json

```diff
@@ -1,45 +1,19 @@
 [
     {
         "name": "Whole Team Investigation",
```

Comment:
Once @mcovalt's examples land in the codebase a bit more solidly, I think it would make sense to create a couple of investigations featuring the speckled test data and then a more realistic one, but I can take care of that in a future PR.